### PR TITLE
Syntax makeover for clear-db-es-contents

### PR DESCRIPTION
**Pull request by netsettler**
* Port some argument changes to clear-db-es-contents from `cgap-portal`.
* Create a `.flake8` file.

### CODE REVIEW

File: src/encoded/tests/test_clear_db_es_contents.py [hunk 1]
```diff
@@ -22,56 +56,254 @@ def test_clear_db_tables(app, testapp):
...
 
-def test_run_clear_db_envs(app, testapp):
+_FOURFRONT_PRODUCTION_ENVS = ['fourfront-production-blue', 'fourfront-production-green', 'data', 'staging']
+# Really we only care about the first of these names, but the rest are names that were at one time
+# planned to be stg or prd names for cgap, so we'll use them to tell that run_clear_db_es is properly
+# skipping any such names. -kmp 4-Jun-2022
+_CGAP_PRODUCTION_ENVS = ['fourfront-cgap', 'fourfront-cgap-green', 'cgap-green', 'fourfront-cgap-blue', 'cgap-blue']
 
-    orig_env = app.registry.settings.get('env.name')
+_PRODUCTION_ENVS = _FOURFRONT_PRODUCTION_ENVS if is_running_app(APP_FOURFRONT) else _CGAP_PRODUCTION_ENVS
```
Comment (willronchetti) [inline src/encoded/tests/test_clear_db_es_contents.py:65]: Probably don't need to do this sort of thing and can just assign the FF names directly.

File: src/encoded/tests/test_clear_db_es_contents.py [hunk 2]
```diff
@@ -22,56 +56,254 @@ def test_clear_db_tables(app, testapp):
...
+                    assert mock_run_create_mapping.call_count == expected_es_clears
+
+                    allowed_envs = ['fourfront-test-env-decoy-1', 'fourfront-test-env-decoy-2']
+                    # test if we are only running on specific envs
+                    with logged_messages(error=[(f'clear_db_es_contents: The current environment, {test_env},'
+                                                 f' is not {disjoined_list(allowed_envs)}.'
+                                                 f' Skipping the attempt to clear DB.')]):
+                        assert run_clear_db_es(app, only_envs=allowed_envs, skip_es=False) is False
+                    expected_db_clears += 0
+                    expected_es_clears += 0
```
Comment (willronchetti) [inline src/encoded/tests/test_clear_db_es_contents.py:200]: Same comment from CGAP - I get being explicit here but still feels unnecessary.

**Review by willronchetti — APPROVED** [review]

File: src/encoded/tests/test_clear_db_es_contents.py (hunk 1 above)
Comment (netsettler) [reply src/encoded/tests/test_clear_db_es_contents.py:65]: I want to share the script between both repos.

File: src/encoded/tests/test_clear_db_es_contents.py (hunk 2 above)
Comment (netsettler) [reply src/encoded/tests/test_clear_db_es_contents.py:200]: If I didn't do this, I'd put the same thing commented out. It seems harmless, so I'm ignoring your uneasiness. :)

File: src/encoded/tests/test_clear_db_es_contents.py (hunk 1 above)
Comment (netsettler) [reply src/encoded/tests/test_clear_db_es_contents.py:65]: I have a plan in the back of my mind to pull these things out to a common repo that is actually shared.

File: src/encoded/tests/test_clear_db_es_contents.py (hunk 2 above)
Comment (netsettler) [reply src/encoded/tests/test_clear_db_es_contents.py:165]: The idea is so visually as you scan down the page, you can read off the number of expected changes for each section. The math is secondary to the self-documentation.

**Comment by netsettler** [comment]
This is passing tests locally on my machine at this point.

**Pull request merged**